src/uwtools/tests/test_rocoto.py

```diff
@@ -86,6 +86,24 @@ def instance(self, assets):
     def root(self):
         return rocoto.Element("root")
 
+    def test__add_compound_time_string_basic(self, instance, root):
+        config = {"foo": "bar"}
+        instance._add_compound_time_string(e=root, config=config, tag="foo")
+        child = root[0]
+        assert child.tag == "foo"
+        assert child.text == "bar"
```

Comment:
This method, and the one below, are direct tests of the `_add_compound_time_string()` method in `_RocotoXML`, first with a basic string value, then with a `cyclestr` block.